### PR TITLE
02. Time-Converter

### PR DESCRIPTION
**Pull request by MilenaGeorgieva95**
02. Time-Converter

### CODE REVIEW

**Review by georgi-georgiev95 — COMMENTED** [review]
Adding the event listener to the main container makes the code difficult to read. U can use querrySelectorAll('button') and then parse it to an Array and using forEach u can add events to the buttons.

**Comment by MilenaGeorgieva95** [comment]
Thank you! I have done it like that also, it says in the task description to do it that way. Just wanted to try another option to see if it works. Just wanted to say also-I am your big fan from the lectures, your code is always on point😁

File: 05-dom-events/exercise/02. Time-Converter/solution.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+function attachEventsListeners() {
+  function attachEventsListeners() {
```
Comment (antoniaat) [inline 05-dom-events/exercise/02. Time-Converter/solution.js]: Why these functions have the same name?

File: 05-dom-events/exercise/02. Time-Converter/solution.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+function attachEventsListeners() {
+  function attachEventsListeners() {
+    let main = document.querySelector("main");
```
Comment (antoniaat) [inline 05-dom-events/exercise/02. Time-Converter/solution.js]: Consider declaring variables that you are not going to change with const

File: 05-dom-events/exercise/02. Time-Converter/solution.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      days: (num) => {
+        return {
+          days: num,
+          hours: num * 24,
+          minutes: num * 1440,
+          seconds: num * 86400,
+        };
+      },
+      hours: (num) => {
+        return {
```
Comment (antoniaat) [inline 05-dom-events/exercise/02. Time-Converter/solution.js]: You can try to omit the return statement with one idea shorter syntax:

 days: (num) => ({
          days: num,
          hours: num * 24,
          minutes: num * 1440,
          seconds: num * 86400, 
}),

**Review by antoniaat — CHANGES_REQUESTED** [review]

**Pull request closed without merging**